cpp/DataStorm/partialUpdate/AtmosphericConditions.ice

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ZeroC, Inc.
+
+module ClearSky
+{
+
+    enum UpdateTag
+    {
+        TemperatureUpdated,
+        HumidityUpdated
+    }
+
+    /// Represents the atmospheric conditions measured by a sensor.
```

Comment:
Unless there is compelling reason to use a struct, I would use a class like all the other AtmosphericConditions.